opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -190,7 +190,8 @@ impl UserEventsExporter {
                 // for each event.
                 // TODO: What if the event name is not provided? "Log" is used as default.
                 // TODO: Should event_tag be non-zero?
-                eb.reset(log_record.event_name().unwrap_or("Log"), 0);
+                let event_name = log_record.event_name().filter(|s| !s.is_empty()).unwrap_or("Log");
```

Comment:
Also, I think it would be better to validate this upstream at SDK level itself in `SdkLogRecord::set_event_name` method. Why allow users to set empty/whitespace only strings at all?

Reply:
we can control it at the appender level - don't set event_name if it's empty string. I am fine to move the fix there we all agree.,

Reply:
in an upcoming release, we can consider validating event_name in the sdk itself.

Reply:
It's better to handle it at the appender level, if we decide to do it at all, since the event is already parsed there. No need to check again in the SDK.

Reply:
@lalitb There are two cases where we would benefit from doing the validation at the SDK level:
- We can only do this for the appenders we host in the main repo. We can't enforce this validation for custom appender implementations
-  Since the `Logger` and `LogRecord` APIs are public, technically a user can emit logs using these APIs directly without having to use any kind of appender.